sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -1494,26 +1488,5 @@ public void testAvroSinkShardedWrite() throws Exception {
     }
     // TODO: for Write only, test withSuffix,
     // withShardNameTemplate and withoutSharding.
-
-    @Test
-    @Category(ValidatesRunner.class)
-    public void testPrimitiveReadDisplayData() {
```

Comment:
This is effectively a duplicate of `AvroIOTest$SimpleTests.testReadDisplayData` and doesn't need to a be a validates runner test since it doesn't run a pipeline.